roboplot/core/debug_movement.py

```diff
@@ -48,7 +48,7 @@ def __init__(self, millimetres_per_step, bgimage_path=None, pixels_per_mm=3):
 
         # Setup image dimensions
         self.pixels_per_mm = pixels_per_mm
-        a4paper = (210, 297)  # (x then y as tuple)
+        a4paper = (210, 297)  # (x then y as numpy arrays take shape as width then height  (num columns, num rows).
```

Comment:
Sorry to come back to this, because it does seem nit-picky, but `numpy` is possibly the source of the y,x convention.
Numpy arrays take and return shape as *height then width*. Indeed, `numpy.zeros((2,3))` has two rows and three columns. It is openCV which asks for image dimensions as *width then height*.